sw/device/silicon_creator/lib/otbn_boot_services.c

```diff
@@ -70,6 +73,32 @@ enum {
   kOtbnBootModeAttestationKeySave = 0x64d,
 };
 
+OT_WARN_UNUSED_RESULT
+static rom_error_t load_attestation_keygen_seed(
+    attestation_key_seed_t additional_seed, uint32_t *seed) {
+  // Set flash page configuration and permissions.
```

Comment:
There are 3 different seeds, one for each attestation keypair (UDS, CDI_0, CDI_1). However, for ES the all live on the same flash info page. I did this because for ES we will generate all keys at the ROM_EXT stage (since we cannot change the ROM now), so all seeds will be read out of flash in the same boot stage. See this [diagram](https://docs.google.com/drawings/d/1ZnlNBCeByk8MdrwKMuLT_zz0bxIiqmI3DEWkHQvzu8Q/edit?usp=sharing). For prod however, I would like to move the first two seeds (for UDS and CDI_1) to a page, and the seed for CDI_2 to a page.

Reply:
ack. SGTM. I agree with the approach for PROD.